pkg/endpoint/bpf.go

```diff
@@ -161,7 +161,7 @@ func (e *Endpoint) writeHeaderfile(prefix string, owner Owner) error {
 	fw.WriteString(common.FmtDefineAddress("LXC_MAC", e.LXCMAC))
 	fw.WriteString(common.FmtDefineAddress("LXC_IP", e.IPv6))
 	if e.IPv4 != nil {
-		fmt.Fprintf(fw, "#define LXC_IPV4 %#x\n", binary.BigEndian.Uint32(e.IPv4))
+		fmt.Fprintf(fw, "#define LXC_IPV4 %#x\n", byteorder.HostToNetworkSliceU32(e.IPv4))
```

Comment:
@tgraf If I make that change then the connectivity tests start failing, not sure why but looking into it 
https://github.com/cilium/cilium/blob/master/tests/01-ct.sh#L99-L102